ui/css/oae.signup.css

```diff
@@ -251,6 +407,7 @@ main > h3 {
 
     #signup-options-divider-horizontal {
         display: block;
+        width: 100%;
     }
 
     #signup-options-local {
```

Comment:
Rather than having the "create account" section display like this:

<img width="813" alt="screen shot 2015-10-06 at 21 37 13" src="https://cloud.githubusercontent.com/assets/109850/10329188/8459d392-6c72-11e5-8690-698613b162a0.png">

We should change it to look like this:

<img width="813" alt="screen shot 2015-10-06 at 21 38 16" src="https://cloud.githubusercontent.com/assets/109850/10329191/9123be3a-6c72-11e5-8402-68629e7f15d1.png">